diff --git a/node_modules/react-katex/node_modules/katex/dist/katex.js b/node_modules/react-katex/node_modules/katex/dist/katex.js
index 2de2003..ae53e66 100644
--- a/node_modules/react-katex/node_modules/katex/dist/katex.js
+++ b/node_modules/react-katex/node_modules/katex/dist/katex.js
@@ -14624,7 +14624,7 @@ const functions = _functions;
  */
 const spaceRegexString = "[ \r\n\t]";
 const controlWordRegexString = "\\\\[a-zA-Z@]+";
-const controlSymbolRegexString = "\\\\[^\uD800-\uDFFF]";
+const controlSymbolRegexString = "\\\\[^\\uD800-\\uDFFF]";
 const controlWordWhitespaceRegexString = "(" + controlWordRegexString + ")" + spaceRegexString + "*";
 const controlSpaceRegexString = "\\\\(\n|[ \r\t]+\n?)[ \r\t]*";
 const combiningDiacriticalMarkString = "[\u0300-\u036f]";
@@ -14635,7 +14635,7 @@ controlSpaceRegexString + "|") +
 "([!-\\[\\]-\u2027\u202A-\uD7FF\uF900-\uFFFF]" + (// single codepoint
 combiningDiacriticalMarkString + "*") +
 // ...plus accents
-"|[\uD800-\uDBFF][\uDC00-\uDFFF]" + (// surrogate pair
+"|[\\uD800-\\uDBFF][\\uDC00-\\uDFFF]" + (// surrogate pair
 combiningDiacriticalMarkString + "*") +
 // ...plus accents
 "|\\\\verb\\*([^]).*?\\4" +
diff --git a/node_modules/react-katex/node_modules/katex/dist/katex.mjs b/node_modules/react-katex/node_modules/katex/dist/katex.mjs
index b4a83b4..dc61e4c 100644
--- a/node_modules/react-katex/node_modules/katex/dist/katex.mjs
+++ b/node_modules/react-katex/node_modules/katex/dist/katex.mjs
@@ -13512,7 +13512,7 @@ var functions = _functions;
  */
 var spaceRegexString = "[ \r\n\t]";
 var controlWordRegexString = "\\\\[a-zA-Z@]+";
-var controlSymbolRegexString = "\\\\[^\uD800-\uDFFF]";
+var controlSymbolRegexString = "\\\\[^\\uD800-\\uDFFF]";
 var controlWordWhitespaceRegexString = "(" + controlWordRegexString + ")" + spaceRegexString + "*";
 var controlSpaceRegexString = "\\\\(\n|[ \r\t]+\n?)[ \r\t]*";
 var combiningDiacriticalMarkString = "[\u0300-\u036f]";
@@ -13523,7 +13523,7 @@ controlSpaceRegexString + "|") +
 "([!-\\[\\]-\u2027\u202A-\uD7FF\uF900-\uFFFF]" + (// single codepoint
 combiningDiacriticalMarkString + "*") +
 // ...plus accents
-"|[\uD800-\uDBFF][\uDC00-\uDFFF]" + (// surrogate pair
+"|[\\uD800-\\uDBFF][\\uDC00-\\uDFFF]" + (// surrogate pair
 combiningDiacriticalMarkString + "*") +
 // ...plus accents
 "|\\\\verb\\*([^]).*?\\4" +
diff --git a/node_modules/react-katex/node_modules/katex/src/Lexer.ts b/node_modules/react-katex/node_modules/katex/src/Lexer.ts
index ad3ac38..694db2d 100644
--- a/node_modules/react-katex/node_modules/katex/src/Lexer.ts
+++ b/node_modules/react-katex/node_modules/katex/src/Lexer.ts
@@ -43,7 +43,7 @@ import type Settings from "./Settings";
  */
 const spaceRegexString = "[ \r\n\t]";
 const controlWordRegexString = "\\\\[a-zA-Z@]+";
-const controlSymbolRegexString = "\\\\[^\uD800-\uDFFF]";
+const controlSymbolRegexString = "\\\\[^\\uD800-\\uDFFF]";
 const controlWordWhitespaceRegexString =
     `(${controlWordRegexString})${spaceRegexString}*`;
 const controlSpaceRegexString = "\\\\(\n|[ \r\t]+\n?)[ \r\t]*";
@@ -54,7 +54,7 @@ const tokenRegexString = `(${spaceRegexString}+)|` +  // whitespace
     `${controlSpaceRegexString}|` +                   // \whitespace
     "([!-\\[\\]-\u2027\u202A-\uD7FF\uF900-\uFFFF]" +  // single codepoint
     `${combiningDiacriticalMarkString}*` +            // ...plus accents
-    "|[\uD800-\uDBFF][\uDC00-\uDFFF]" +               // surrogate pair
+    "|[\\uD800-\\uDBFF][\\uDC00-\\uDFFF]" +           // surrogate pair
     `${combiningDiacriticalMarkString}*` +            // ...plus accents
     "|\\\\verb\\*([^]).*?\\4" +                       // \verb*
     "|\\\\verb([^*a-zA-Z]).*?\\5" +                   // \verb unstarred
