diff --git a/factory.js b/factory.js
index 40f3edccbebc45b491dd3488ed619ec9208be8d9..47971e2f49442fc0a1a571c96ef4cecd07bf37be 100644
--- a/factory.js
+++ b/factory.js
@@ -315,4 +315,9 @@ function plotComponentFactory(Plotly) {
   };
   return PlotlyComponent;
 }
+module.exports = plotComponentFactory;
+module.exports.default = plotComponentFactory;
+Object.defineProperty(module.exports, "__esModule", {
+  value: true
+});
 //# sourceMappingURL=factory.js.map
\ No newline at end of file
diff --git a/plot2d.js b/plot2d.js
new file mode 100644
index 0000000000000000000000000000000000000000..4152081a2339c4bf638ed02850b6c798b030a6f7
--- /dev/null
+++ b/plot2d.js
@@ -0,0 +1,25 @@
+"use strict";
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+exports["default"] = void 0;
+
+var _factory = _interopRequireDefault(require("./factory"));
+
+var _plotly = _interopRequireDefault(require("plotly.js/lib/core"));
+
+var _scatter = _interopRequireDefault(require("plotly.js/lib/scatter"));
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }
+
+_plotly["default"].register([_scatter["default"]]);
+
+var PlotComponent = (0, _factory["default"])(_plotly["default"]);
+var _default = PlotComponent;
+exports["default"] = _default;
+module.exports = _default;
+module.exports.default = _default;
+Object.defineProperty(module.exports, "__esModule", {
+  value: true
+});
diff --git a/plot3d.js b/plot3d.js
new file mode 100644
index 0000000000000000000000000000000000000000..b5026349834b11d6dcc6a848858f4598b18a75e7
--- /dev/null
+++ b/plot3d.js
@@ -0,0 +1,27 @@
+"use strict";
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+exports["default"] = void 0;
+
+var _factory = _interopRequireDefault(require("./factory"));
+
+var _plotly = _interopRequireDefault(require("plotly.js/lib/core"));
+
+var _scatter3d = _interopRequireDefault(require("plotly.js/lib/scatter3d"));
+
+var _mesh3d = _interopRequireDefault(require("plotly.js/lib/mesh3d"));
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }
+
+_plotly["default"].register([_scatter3d["default"], _mesh3d["default"]]);
+
+var PlotComponent = (0, _factory["default"])(_plotly["default"]);
+var _default = PlotComponent;
+exports["default"] = _default;
+module.exports = _default;
+module.exports.default = _default;
+Object.defineProperty(module.exports, "__esModule", {
+  value: true
+});
diff --git a/react-plotly.js b/react-plotly.js
index f599b9ed1120feb71747a34088869e429a54a086..f1d6daf896049c040c35aceeef2b6b0aa4954206 100644
--- a/react-plotly.js
+++ b/react-plotly.js
@@ -14,4 +14,9 @@ function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "d
 var PlotComponent = (0, _factory["default"])(_plotly["default"]);
 var _default = PlotComponent;
 exports["default"] = _default;
+module.exports = _default;
+module.exports.default = _default;
+Object.defineProperty(module.exports, "__esModule", {
+  value: true
+});
 //# sourceMappingURL=react-plotly.js.map
